frontend/src/app/applications/permit-application-list/permit-application-list.component.html

```diff
@@ -0,0 +1,85 @@
+<section class="usa-section">
+  <div class="usa-grid">
+    <h1>Mount Baker-Snoqualmie National Forest</h1>
+    <div class="application-status-toggle-wrapper">
+      <label for="application-status-toggle">Filter by</label>
+      <select class="usa-input-inline" id="application-status-toggle" (change)="applicationStatusChange($event)">
```

Comment:
Would it make sense to create a status component or service that other controllers can leverage for this?

Reply:
I'll create an issue for this.